datahub/data.py

```diff
@@ -51,6 +53,7 @@
 
 opal_df = create_opal_frame()
 dsr_data: list[dict[str, str | list]] = []  # type: ignore[type-arg]
+wesim_data: dict[str, pd.DataFrame] = {}
```

Comment:
This type is probably wrong now
```suggestion
wesim_data: dict[str, dict] = {}
```